src/voice/connection.rs

```diff
@@ -602,7 +604,7 @@ impl Connection {
 
         let crypted = {
             let slice = &packet[HEADER_LEN..HEADER_LEN + len];
-            secretbox::seal(slice, &nonce, &self.key)
+            self.cipher.encrypt(&nonce, slice).unwrap()
```

Comment:
Could you please change this to an `expect` if encryption is never actually able to fail?
```suggestion
            self.cipher.encrypt(&nonce, slice)
                .expect("[Voice] Couldn't encrypt packet.")
```
Looking at `aead`'s `Error` type doesn't reveal much about this, so I expect you know more (i.e., if this is only "fallible" to fit the `Aead` interface). Adding a variant to "voice/error.rs" and using `?` would be a good idea if otherwise.

Reply:
There's presently only one possible runtime error in the current implementation:

https://github.com/RustCrypto/AEADs/blob/master/xsalsa20poly1305/src/lib.rs#L286-L289

The current implementation can't trigger it, and in short, you'd really have to go out of your way to trigger it. If you ever did, encryption would always fail 100% of the time.

That said, I can still change it to `expect` if you like for futureproofing.